### PR TITLE
Removed login url per sector

### PR DESCRIPTION
**Pull request by tom-schier**
- Removed to sector specific login
- Build new containers and published

### CODE REVIEW

File: test-data-server/src/app.ts [hunk 1]
```diff
@@ -913,14 +913,16 @@ app.post(`${basePath}/energy/accounts/billing`, async (req: Request, res: Respon
...
     try {
         console.log(`Received request on ${port} for ${req.url}`);
-        if (sectorIsValid(req.params?.sector) == false) {
+        let qry: any = req.query
+        let customers = await dbService.getLoginInformation(req.params?.sector, qry["loginId"])
+        if (customers == null) {
+            console.log(`Error: customer not found ${req.params?.loginId}`);
             res.status(404).json('Not Found');
-            return;
+            return;
```
Comment (vadkor) [inline test-data-server/src/app.ts]: I hate trailing blanks.

File: test-data-server/src/services/database.interface.ts [hunk 2]
```diff
@@ -58,7 +58,7 @@ export interface IDatabase {
 
     getUserForLoginId(loginId: string, userType: string): Promise<string| undefined>;
 
-    getLoginInformation(sector: string): Promise<CustomerModel[] | undefined>;
+    getLoginInformation(sector: string, loginId?: string): Promise<CustomerModel[] | undefined>;
```
Comment (vadkor) [inline test-data-server/src/services/database.interface.ts]: Shouldn't it be
```suggestion
    getLoginInformation(sector?: string, loginId?: string): Promise<CustomerModel[] | undefined>;
```
?

File: test-data-server/src/services/single.service.ts [hunk 3]
```diff
@@ -670,51 +670,50 @@ export class SingleData implements IDatabase {
     }
 
     // get all the logins for the ACCC cdr-auth-server UI
-    async getLoginInformation(sector: string): Promise<CustomerModel[] | undefined> {
+    async getLoginInformation(sector: string, loginId: string): Promise<CustomerModel[] | undefined> {
```
Comment (vadkor) [inline test-data-server/src/services/single.service.ts]: And here:
```suggestion
    async getLoginInformation(sector?: string, loginId: string): Promise<CustomerModel[] | undefined> {
```

File: test-data-server/src/services/single.service.ts [hunk 4]
```diff
@@ -670,51 +670,50 @@ export class SingleData implements IDatabase {
...
                             MaskedName: acc?.account?.maskedNumber,
-                            DisplayName: acc?.account?.displayName
+                            DisplayName: `Energy - ${acc?.account?.displayName}`
                         };
                         accounts.push(loginAccount)
                     })
                     aModel.Accounts = accounts;
                     loginModel.push(aModel);
 
+                    // TODO Once we have to API implemented for Banking, we can uncomment this
```
Comment (vadkor) [inline test-data-server/src/services/single.service.ts]: Typo - to -> the?

File: test-data-server/src/app.ts (hunk 1 above)
Comment (tom-schier) [reply test-data-server/src/app.ts]: Removed

File: test-data-server/src/services/database.interface.ts (hunk 2 above)
Comment (tom-schier) [reply test-data-server/src/services/database.interface.ts]: Updated

**Review by vadkor — APPROVED** [review]

**Comment by tom-schier** [comment]
Closes #12

**Pull request merged**